corehq/apps/userreports/models.py

```diff
@@ -262,8 +262,9 @@ def __unicode__(self):
     @memoized
     def config(self):
         try:
-            return DataSourceConfiguration.get(self.config_id)
-        except ResourceNotFound:
+            from corehq.apps.userreports.views import get_datasource_config_or_404
```

Comment:
need for an inline import makes me think these functions are in the wrong file, but this fixes the errors for now.